distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc

```diff
@@ -35,12 +35,12 @@ If console access is needed while running as a service, run the `client` script:
 
 .*NIX
 ----
-<${branding}_HOME/bin/client
+<${branding}_HOME>/bin/client
 ----
 
 .Windows
 ----
-<${branding}_HOME>/bin/client.bat
+<${branding}_HOME>/bin/client.bat -h localhost
```

Comment:
Good question. The reason I had to put `-h localhost` on Windows is because there's a bug in the Karaf's batch file and the script doesn't work with no arguments. If I add a note, I'd probably have to also explain how to use the argument on both platforms and mention that it is required on Windows or else it won't work. Just afraid that it'll make things more messy than really needed here. Thoughts?